zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -830,6 +832,22 @@ private String summarizeCommandDistribution(final Record<?> record) {
         .toString();
   }
 
+  private String summarizeMessageCorrelation(final Record<?> record) {
+    final var value = (MessageCorrelationRecordValue) record.getValue();
+
+    final var result = new StringBuilder().append("\"").append(value.getName()).append("\"");
+
+    if (!StringUtils.isEmpty(value.getCorrelationKey())) {
+      result.append(" correlationKey: ").append(value.getCorrelationKey());
+    }
+
+    result.append(" processInstanceKey: ").append(value.getProcessInstanceKey());
+
+    result.append(summarizeVariables(value.getVariables()));
+
+    return result.toString();
+  }
+
```

Comment:
🔧 This method can be refactored like that:

```suggestion
  private String summarizeMessageCorrelation(final Record<?> record) {
    final MessageCorrelationRecordValue value = (MessageCorrelationRecordValue) record.getValue();
    final String correlationKey = value.getCorrelationKey();

    StringBuilder result = new StringBuilder()
        .append("\"").append(value.getName()).append("\"");

    if (correlationKey != null && !correlationKey.isEmpty()) {
        result.append(" correlationKey: ").append(correlationKey);
    }

    result.append(" processInstanceKey: ").append(value.getProcessInstanceKey())
          .append(summarizeVariables(value.getVariables()));

    return result.toString();
}

```